experimental/apps-mcp/lib/prompts/apps.tmpl

```diff
@@ -9,6 +9,21 @@
 DATABRICKS APPS DEVELOPMENT
 ============================
 
+When building or planning a new Databricks app, use the following steps:
+
+1. MUST start by scaffolding a new Databricks app using command: invoke_databricks_cli 'experimental apps-mcp tools init-template --name my-app-name --description "My app description"'
```

Comment:
Ideally we should separate the concerns: app.tmpl is about the apps flow (init-template, validate, deploy when asked) only but agnostic to template details. It even may not know the stack at this point. 

Ok for now, but is subject for restructuring.